Asymmetrik/react-starter
# Feat/support customer manager kms secrets

## Acceptance Criteria
A container can use secrets encrypted with a customer managed key instead of an aws managed key

## Change Description
A container can use secrets encrypted with a customer managed key instead of an aws managed key

## Verification Instructions
Deploy the fargate service with some secrets. Have the secrets module create a new key and pass the arn to the task definition to update the permissions. A container should be able to start 

## Commit Message
feat: Customer managed keys now supported by the fargate service
